src/main/kotlin/kweb/WebBrowser.kt

```diff
@@ -59,37 +65,79 @@ class WebBrowser(private val sessionId: String, val httpRequestInfo: HttpRequest
         }
     }
 
-    fun execute(js: String) {
-        kweb.execute(sessionId, js)
+    data class JSFunction(val js: String, val params: String)
+    //this function substitutes "{}" in user supplied javascript, for randomly generated variable names
+    private fun getJsFunction(rawJs: String): JSFunction {
+        val rng = Random()
+        var js = rawJs
+        val params = StringBuilder()
+        while (js.contains("{}")) {
+            //a few random letters, an underscore, and a randomly generated number should make a variable name that no one would ever come up with
+            val jsVarName = "rzd_${rng.nextInt(1000)}"
```

Comment:
Might want to increase to nextInt(100000) to reduce chance of collisions.

Reply:
Alternatively we could use code similar to WebBrowser.generateId() (but we probably don't want to use generateId() itself because it's useful for element ids to be sequential)